apiv5/README.md

```diff
@@ -24,7 +24,11 @@ export KTAPI_AUTH_EMAIL=<your kentik api credentials email>
 export KTAPI_AUTH_TOKEN=<your kentik api credentials token>
 
 # Run from the repository root
-go test -tags examples -count 1 -v ./examples/users_example_test.go
+./tools/test.sh -v apiv5 -t TestDemonstrateUsersGetAll
```

Comment:
As discussed offline:
- please pass all positional arguments as test args passed to `go test` calls
- do not run examples on CI
- there is no need for -v and -t options if we allow user to pass test args